layout/_third-party/share/needsharebutton/floatdom.swig

```diff
@@ -0,0 +1 @@
+<div id="need-share-button-float"><i class="fa fa-share-alt" aria-hidden="true"></i></div>
```

Comment:
And can u please to split this 2 files into 1?

Reply:
It is ok to use macro to identify whether it is float or postbottom?And where should i import the macro in _layout.swig and post.swig?

Reply:
U joking? This is floatdom.**swig**

Reply:
I will make a new file called dom.swig like this:
```swig
{% macro render type %}
    <span id="needsharebutton-{{type}}"><i class="fa fa-share-alt" aria-hidden="true"></i></span>
{% endmacro %}
```
(maybe i reply in the wrong place...)

Reply:
Seems good. But why filename `dom.swig`?